ts/screens/authentication/cie/CiePinScreen.tsx

```diff
@@ -136,20 +139,27 @@ const CiePinScreen = () => {
           cieConsentUri: loginUri
         });
       } else {
-        navigation.navigate(ROUTES.CIE_CARD_READER_SCREEN, {
-          ciePin: pin,
-          authorizationUri: authUrlGenerated
-        });
+        if (isNfcEnabled) {
```

Comment:
Since both screens have the same params, instead of an if-else statement, a ternary operator could be used that only affects the screen selection. 
For example:
```tsx
 navigation.navigate(isNfcEnabled  ? ROUTES.CIE_CARD_READER_SCREEN : ROUTES.CIE_ACTIVATE_NFC_SCREEN, {
      ciePin: pin,
      authorizationUri: authUrlGenerated
  });
```
What do you think?